airflow/settings.py

```diff
@@ -61,28 +32,38 @@ def gauge(cls, stat, value, rate=1, delta=False):
  """
 
 BASE_LOG_URL = '/admin/airflow/log'
-AIRFLOW_HOME = os.path.expanduser(conf.get('core', 'AIRFLOW_HOME'))
-SQL_ALCHEMY_CONN = conf.get('core', 'SQL_ALCHEMY_CONN')
 LOGGING_LEVEL = logging.INFO
-DAGS_FOLDER = os.path.expanduser(conf.get('core', 'DAGS_FOLDER'))
```

Comment:
These should be accessed through the configuration convenience methods to make sure they correspond to the current config.